MAUI/Cartesian-Charts/StackedLine.md

```diff
@@ -0,0 +1,445 @@
+                                  YBindingPath="Value"  
+                                  StrokeDashArray="{StaticResource dashArray}"/>
+
+        <chart:StackingLineSeries ItemsSource="{Binding Data2}"
+                                  XBindingPath="Month"
+                                  YBindingPath="Value"  
+                                  StrokeDashArray="{StaticResource dashArray}"/>
+
+        <chart:StackingLineSeries ItemsSource="{Binding Data3}"
+                                  XBindingPath="Month"
+                                  YBindingPath="Value"   
+                                  StrokeDashArray="{StaticResource dashArray}"/>
+
+        <chart:StackingLineSeries ItemsSource="{Binding Data4}"
+                                  XBindingPath="Month"
+                                  YBindingPath="Value"       
+                                  StrokeDashArray="{StaticResource dashArray}"/>
+    </chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCartesianChart chart = new SfCartesianChart();
+    CategoryAxis primaryAxis = new CategoryAxis();
+    chart.XAxes.Add(primaryAxis);
+    NumericalAxis secondaryAxis = new NumericalAxis();
+    chart.YAxes.Add(secondaryAxis);
+
+    ViewModel viewModel = new ViewModel();
+    
+    DoubleCollection doubleCollection = new DoubleCollection();
+    doubleCollection.Add(5);
+    doubleCollection.Add(2);
+    . . .
+    StackingLineSeries series1 = new  StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data1,
+        StrokeDashArray = doubleCollection
+    };
+
+    StackingLineSeries series2 = new StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data2,
+        StrokeDashArray = doubleCollection
+    };
+
+    StackingLineSeries series3 = new StackingLineSeries()       
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data3,
+        StrokeDashArray = doubleCollection
+    };
+
+    StackingLineSeries series4 = new StackingLineSeries()
+    {
+        XBindingPath = "Month",
+        YBindingPath = "Value",
+        ItemsSource = viewModel.Data4,
+        StrokeDashArray = doubleCollection
+    };
+
+    chart.Series.Add(series1);
+    chart.Series.Add(series2);     
+    chart.Series.Add(series3); 
+    chart.Series.Add(series4); 
+
+    this.Content = chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Dashed Stacked Line chart in MAUI](Chart-types-images/maui_dashed_stacked_line_chart.png)
+
+## Enable Marker
+
+A marker, also known as a symbol, is used to determine or highlight the position of the data point. To enable markers in the series, set the [ShowMarkers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.StackingLineSeries.html#Syncfusion_Maui_Charts_StackingLineSeries_ShowMarkers) property to `true`.
```

Comment:
The StackingLineSeries has not been published live yet. Why is the API link included here? How does it work?

Reply:
removed link